lib/MarkupTests/MarkupTests.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  MarkupTests.swift
+//  MarkupTests
+//
+//  Created by Miguel Ángel Díaz on 22/02/2019.
+//  Copyright © 2019 47 Degrees. All rights reserved.
+//
+
+import XCTest
+@testable import Markup
+
+
+//case nef(command: Nef.Command, [Node])
+//case markup(title: String?, String)
+//case comment(String)
+//case code(String)
+//case unknown(String)
+
+
+
+class MarkupTests: XCTestCase {
+
+    override func setUp() {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+
+    func testPlainPlayground_parse_returnsCodeNode() {
+        let input = "import Bow // testing\n"
+        let expected: [Node] = [.code("import Bow // testing\n")]
+
+        let result = Markup.SyntaxAnalyzer.parse(content: input)
+
+        XCTAssertEqual(result, expected)
+    }
+
+    func testPlainPlaygroundWithSimpleMarkup_parse_returnsMarkupNode() {
+        let input = "//: This is a test\n"
+        let expected: [Node] = [.markup(description: nil, "This is a test\n")]
+
+        let result = Markup.SyntaxAnalyzer.parse(content: input)
+
+        XCTAssertEqual(result, expected)
+    }
+//
+//    func testLeftDelimiterWithoutRightDelimiter_parse_returnsPlainText() {
```

Comment:
Remove commented code.

Reply:
Yep, sorry, I have not finished this - I have to complete the rest of the unit-tests